TorchSharp/Raw/Blas.tt

```diff
@@ -0,0 +1,158 @@
+﻿<#@ Template Language="C#" #>
+<#@ Assembly Name="System.Core" #>
+<#@ Output Extension=".generated.cs" #>
+<#@ Include File="Types.ttinclude" #>
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace TorchSharp {
+<#
+foreach (var type in TorchTypeDef.Types) {
+#>
+
+    public partial class <#=type.Name#>Tensor : IDisposable {
+
+        // *** Level 1
+
+        // Swap x and y.

```

Comment:
I think this should go into summary, no?

Reply:
Unless you put them in // so that you can add the summary into the actual C# API. Is this your plan?

Reply:
@interesaaat I plan to have a short description like this one both here and in xmldoc for high-level API. That is, when you look at the code, you have a rough idea what the function is for, yet we don't clutter the generated docs with low-level  API that we don't expose anyway.